### PR TITLE
[DOCS-6628] - Add theme information

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/model/interfaces.md [hunk 1]
```diff
@@ -36,6 +36,92 @@ To create a user interface:
...
+	       "textColor": "#672ed9",
+	       "backgroundColor": "#c7b4ed",
+	       "baseFontSize": "19px",
+	       "fontFamily": "cursive"
+         }
+```
+
+Once you have deployed your application using the admin app, the values from the the `theme` javascript are used for generating the `custom.css` file. If using Docker compose the `custom.css` file is generated during the Docker startup process. The `custom.css` file is used for altering the application theme and the new theme can be found under the `customCssPath` value.
+
+### Generate theme manually
```
Comment (BSekula) [inline process-automation/latest/model/interfaces.md:89]: Not sure if we need to add this info:

`Generate theme manually`, `Generate custom theme locally`

@jesty what do you think?

File: process-automation/latest/model/interfaces.md [hunk 2]
```diff
@@ -36,6 +36,92 @@ To create a user interface:
...
+For example:
+
+```javascript
+"theme": {
+           "primaryColor": "#d92ea8",
+	       "accentColor": "#2eb7d9",
+	       "textColor": "#672ed9",
+	       "backgroundColor": "#c7b4ed",
+	       "baseFontSize": "19px",
+	       "fontFamily": "cursive"
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md:83]: cheers, what do you mean?

[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md:83]: @BSekula

File: process-automation/latest/model/interfaces.md [hunk 3]
```diff
@@ -36,6 +36,92 @@ To create a user interface:
...
+
+The properties of the UI theme are:
+
+> **Note:** See example image below.
+
+| Property | Description |
+| ----------- |----------- |
+| Primary color | The color is used to hightlight important parts of your application, for example the text for the **Personal Files** link in the left pane. |
+| Accent color | The color is used to accent highlight areas of your UI and make parts of it stand out more, for example it can be used for floating action buttons, selection controls such as sliders and switches, highlighting selected text or Progress bars, and some links. |
+| Text color | The color of text that is not as important as other text, for example the text **File Libraries** on the left pane. |
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md:61]: Changed to, The default text color used for the application.
Cheers

File: process-automation/latest/model/interfaces.md [hunk 4]
```diff
@@ -36,6 +36,92 @@ To create a user interface:
...
+
+You can add a theme to the Digital Workspace. There are several different ways of doing this and how you do it will depend on the type of user you are.
+
+### Use the modelling application to change the theme
+
+To change the theme using the **UI** properties in the modelling application.  
+
+1. Sign into the modeling application.
+
+2. Select the **UI** you want to change and enter the new configuration properties under the **Theme** heading.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md:49]: Cheers.

Changed to    > **Note:** You can only change the **Theme** of the **Default UI**.

File: process-automation/latest/model/interfaces.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (BSekula) [reply process-automation/latest/model/interfaces.md:83]: Lines should be aligned :)

```
"theme": {
   "primaryColor": "#d92ea8",
   "accentColor": "#2eb7d9",
   "textColor": "#672ed9",
   ....
```

File: process-automation/latest/model/interfaces.md (hunk 1 above)
Comment (jesty) [reply process-automation/latest/model/interfaces.md:89]: I agree, we could remove, it's too technically

Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md:89]: Removed thanks

File: process-automation/latest/model/interfaces.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md]: changed cheers

File: process-automation/latest/model/interfaces.md [hunk 5]
```diff
@@ -36,6 +36,54 @@ To create a user interface:
 
 4. Enter a name and optional description.
 
+## Theme
+
+You can add a theme to the Digital Workspace. There are several different ways of doing this and how you do it will depend on the type of user you are.
```
Comment (BSekula) [inline process-automation/latest/model/interfaces.md:41]: Now there is only one way :) i.e. using modelin app

**Review by BSekula — APPROVED** [review]

File: process-automation/latest/model/interfaces.md (hunk 5 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/interfaces.md:41]: Removed the entire sentence.

**Review by BSekula — APPROVED** [review]

**Review by Amy-Hyland — COMMENTED** [review]
LGTM

**Review by Amy-Hyland — APPROVED** [review]

**Pull request merged**